patches/api/0422-Deprecate-and-replace-methods-with-old-StructureType.patch

```diff
@@ -0,0 +1,146 @@
++    }
++    // Paper end
+ 
+     /**
+      * Reloads the server, refreshing settings and plugin information.
+diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
+index 6b72eccdcb6f75534a4267a1dd0a4cc2f39e917b..58f1bb9f683c7cba5c39fd6c810e7bbd825ff1ae 100644
+--- a/src/main/java/org/bukkit/Server.java
++++ b/src/main/java/org/bukkit/Server.java
+@@ -775,7 +775,9 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
+      *
+      * @see World#locateNearestStructure(org.bukkit.Location,
+      *      org.bukkit.StructureType, int, boolean)
++     * @deprecated use {@link #createExplorerMap(World, Location, org.bukkit.generator.structure.StructureType, org.bukkit.map.MapCursor.Type)}
+      */
++    @Deprecated // Paper
+     @NotNull
+     public ItemStack createExplorerMap(@NotNull World world, @NotNull Location location, @NotNull StructureType structureType);
+ 
+@@ -796,9 +798,53 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
+      *
+      * @see World#locateNearestStructure(org.bukkit.Location,
+      *      org.bukkit.StructureType, int, boolean)
++     * @deprecated use {@link #createExplorerMap(World, Location, org.bukkit.generator.structure.StructureType, org.bukkit.map.MapCursor.Type, int, boolean)}
+      */
++    @Deprecated // Paper
+     @NotNull
+     public ItemStack createExplorerMap(@NotNull World world, @NotNull Location location, @NotNull StructureType structureType, int radius, boolean findUnexplored);
++    // Paper start
++    /**
++     * Create a new explorer map targeting the closest nearby structure of a
++     * given {@link org.bukkit.generator.structure.StructureType}.
++     * <br>
++     * This method uses implementation default values for radius and
++     * findUnexplored (usually 100, true).
++     *
++     * @param world the world the map will belong to
++     * @param location the origin location to find the nearest structure
++     * @param structureType the type of structure to find
++     * @param mapIcon the map icon to use on the map
++     * @return a newly created item stack or null if it can't find a location
++     *
++     * @see World#locateNearestStructure(org.bukkit.Location,
++     *      org.bukkit.generator.structure.StructureType, int, boolean)
++     */
++    default @Nullable ItemStack createExplorerMap(@NotNull World world, @NotNull Location location, @NotNull org.bukkit.generator.structure.StructureType structureType, @NotNull org.bukkit.map.MapCursor.Type mapIcon) {
++        return this.createExplorerMap(world, location, structureType, mapIcon, 100, true);
++    }
++
++    /**
++     * Create a new explorer map targeting the closest nearby structure of a
++     * given {@link org.bukkit.generator.structure.StructureType}.
++     * <br>
++     * This method uses implementation default values for radius and
++     * findUnexplored (usually 100, true).
```

Comment:
This doesn't seems to be true here, it use the parameters and not the default values. It's also the case for the old methods and the method in the other class